cypress/support/e2e.js

```diff
@@ -0,0 +1 @@
+import 'cypress-axe';
```

Comment:
```suggestion
import 'cypress-axe';

```

Recomenda-se deixar uma linha em branco ao final de cada arquivo para que ferramentas como o GitHub não exibam o ícone de `No newline at end of file`.

Isso é uma boa prática e convenção a ser seguida.